src/MeetingsSDKAdapter.js

```diff
@@ -194,6 +211,116 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
     });
   }
 
+  /**
+   * Attempts to mute the microphone of the given meeting ID.
+   * If the microphone is successfully muted, an audio mute event is dispatched.
+   *
+   * @param {string} ID ID of the meeting to mute audio
+   * @memberof MeetingsSDKAdapter
+   */
+  muteAudioMeeting(ID) {
+    const sdkMeeting = this.fetchMeeting(ID);
+
+    sdkMeeting
+      .muteAudio()
+      .then(() => {
+        sdkMeeting.emit(EVENT_MEDIA_LOCAL_MUTE, {type: MEDIA_TYPE_LOCAL_MUTE_AUDIO});
+      })
+      .catch((error) => {
+        // eslint-disable-next-line no-console
+        console.error(`Unable to mute audio for meeting "${ID}"`, error);
+      });
+  }
+
+  /**
+   * Returns an observable that emits the display data of a mute meeting audio control.
+   *
+   * @param {string} ID ID of the meeting to mute audio
+   * @returns {Observable.<MeetingControlDisplay>}
+   * @memberof MeetingJSONAdapter
+   */
+  muteAudioControl(ID) {
+    const sdkMeeting = this.fetchMeeting(ID);
+    const displayData = {
+      ID: MUTE_AUDIO_CONTROL,
+      text: 'Mute meeting audio',
+      tooltip: 'Mute meeting audio',
+      state: MeetingControlState.INACTIVE,
+    };
+
+    const getDisplayData$ = Observable.create((observer) => {
+      if (sdkMeeting) {
+        observer.next(displayData);
+      } else {
+        observer.error(new Error(`Could not find meeting with ID "${ID}" to mute audio on`));
+      }
+
+      observer.complete();
+    });
+
+    const muteEvent$ = fromEvent(sdkMeeting, EVENT_MEDIA_LOCAL_MUTE).pipe(
+      filter((event) => event.type === MEDIA_TYPE_LOCAL_MUTE_AUDIO),
+      map(() => ({...displayData, state: MeetingControlState.ACTIVE}))
+    );
+
+    return concat(getDisplayData$, muteEvent$);
+  }
+
+  /**
+   * Attempts to mute the camera of the given meeting ID.
+   * If the camera is successfully muted, a video mute event is dispatched.
+   *
+   * @param {string} ID ID of the meeting to mute video
+   * @memberof MeetingsSDKAdapter
+   */
+  muteVideoMeeting(ID) {
+    const sdkMeeting = this.fetchMeeting(ID);
+
+    sdkMeeting
+      .muteVideo()
+      .then(() => {
+        sdkMeeting.emit(EVENT_MEDIA_LOCAL_MUTE, {type: MEDIA_TYPE_LOCAL_MUTE_VIDEO});
```

Comment:
Can you just send the control instead of creating a type? 
```suggestion
        sdkMeeting.emit(EVENT_MEDIA_LOCAL_MUTE, {type: MUTE_VIDEO_CONTROL});
```

OR

```suggestion
        sdkMeeting.emit(EVENT_MEDIA_LOCAL_MUTE, {control: MUTE_VIDEO_CONTROL});
```